Add information about Export customers accounts

EvheniiaKolkhiieva opened the pull request:
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) adds information about Export customers accounts

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/customers/customer-account-manage.html

BarnyShergold commented on src/customers/customer-account-manage.md:
```diff
@@ -99,6 +99,20 @@ See [Managing Subscribers]({% link marketing/newsletter-subscribers.md %}) for m
+
+1. On the _Admin_ sidebar, go to **Customers** > **All Customers**.
+
+1. In the action menu, click Export and select the desired format:
```
I don't think it should be termed "action menu" - this might get confused with the "Action" dropdown on the left of the grid. What about Table Header Menu?

BarnyShergold commented on src/customers/customer-account-manage.md:
```diff
@@ -99,6 +99,20 @@ See [Managing Subscribers]({% link marketing/newsletter-subscribers.md %}) for m
+1. Click <span class="btn">OK</span>.
+
+1. Open the exported file in the browser downloads section.
+
```
There should be a note that this will export EVERY customer. If they want to export a limited number of accounts, they should either tick the ones they want to export OR use the filters to select a range of customer accounts and then use the export.

EvheniiaKolkhiieva commented:
Thank you @BarnyShergold , I have replaced the "Action" with "Table Header Menu" and added the tip about limited customer export. Could you please take a look?

BarnyShergold commented on src/customers/customer-account-manage.md:
```diff
@@ -99,6 +99,23 @@ See [Managing Subscribers]({% link marketing/newsletter-subscribers.md %}) for m
+1. Open the exported file in the browser downloads section.
+
+{:.bs-callout-info}
+The above instruction will export all customer's accounts. If you want to export a limited number, you should either with checkboxes for accounts you want to export, or use filters on the control panel to select a range of customer accounts and then use **Export**.
```
Note : Customer's is possessive singular
```suggestion
The above instruction will export all customer accounts. If you want to export a limited number, you should either use checkboxes for accounts you want to export, or use filters on the control panel to select a range of customer accounts and then use **Export**.
```

BarnyShergold reviewed (changes requested):
Good update - just a couple of grammar issues.

EvheniiaKolkhiieva commented:
Hi @BarnyShergold , thank you! Done ✅

BarnyShergold reviewed (approved):
Excellent! Thanks!

magento-engcom-team commented:
@dobooth the pull request successfully imported.
